src/lucky_web/tags/input_helpers.cr

```diff
@@ -9,6 +9,12 @@ module LuckyWeb::InputHelpers
     })
   end
 
+  def checkbox(field : LuckyRecord::AllowedField, **html_options, unchecked_value : (String | Nil) = nil)
```

Comment:
You can do the equivalent of `(String | Nil)` by using `String?`

Reply:
Also I wonder if we should also have a `checked_value` option as well?

Reply:
`checked_value` would currently be passed as `value`. This is the same as the other inputs. My thoughts where to have a unified api there for the helpers.

Reply:
Oh duh. That makes total sense 👍

Reply:
`String | Nil` => `String?` … the wonders of Crystal. Thanks for pointing that out 👍